src/cdk/commands/renderGraph.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getLogger } from '../../shared/logger'
+import * as telemetry from '../../shared/telemetry/telemetry'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { Commands } from '../../shared/vscode/commands'
+import { Window } from '../../shared/vscode/window'
+import { ConstructNode } from '../explorer/nodes/constructNode'
+import { showErrorWithLogs } from '../../shared/utilities/messages'
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { getCfnDefinitionForStateMachine } from '../explorer/nodes/getCfnDefinition'
+
+/**
+ * Renders a state graph of the state machine represented by the given node.
+ *
+ */
+export async function renderGraphCommand(
+    node: ConstructNode,
+    window = Window.vscode(),
+    commands = Commands.vscode()
+): Promise<AslVisualizationCDK | undefined> {
+    getLogger().debug('Render graph called for: %O', node)
+
+    const uniqueIdentifier = node.label
+    var cdkOutPath = node.id?.replace(`/tree.json/${node.tooltip}`, ``)!
+    var stackName = node.tooltip?.replace(`/${node.label}`, ``)!
+
+    getLogger().info(`Rendering graph: ${uniqueIdentifier}`)
+    try {
+        const cfnDefinition = getCfnDefinitionForStateMachine(uniqueIdentifier, cdkOutPath, stackName)
+
+        const newVisualization = new AslVisualizationCDK(cfnDefinition ? cfnDefinition : 'error', uniqueIdentifier)
+
+        getLogger().info('Rendered graph: %O', uniqueIdentifier)
+        window.showInformationMessage(localize('AWS.cdk.renderGraph.success', 'Rendered graph {0}', uniqueIdentifier))
+        //I know I need to change this telemetry part but not sure how
+        telemetry.recordS3CreateBucket({ result: 'Succeeded' })
```

Comment:
Telemetry is how we record metrics from the Toolkit. For now let's remove all the telemetry references from the code and we can add it back properly in a future commit.